pfio/v2/s3.py

```diff
@@ -447,8 +459,9 @@ def rename(self, src, dst):
 
         '''
         self._checkfork()
-        source = {'Bucket': self.bucket, 'Key': os.path.join(self.cwd, src)}
+        source = {'Bucket': self.bucket, 'Key': _normalize_key(os.path.join(self.cwd, src))}
```

Comment:
This line is too long and fails in flake8. https://ci.preferred.jp/chainerio/79933/#L2274

```
pfio/v2/s3.py:462:80: E501 line too long (92 > 79 characters)
```